chapters/VK_EXT_debug_utils.txt

```diff
@@ -52,6 +56,19 @@ include::{generated}/api/protos/vkSetDebugUtilsObjectNameEXT.txt[]
     pname:pNameInfo->objectType must: not be ename:VK_OBJECT_TYPE_UNKNOWN
   * [[VUID-vkSetDebugUtilsObjectNameEXT-pNameInfo-02588]]
     pname:pNameInfo->objectHandle must: not be dlink:VK_NULL_HANDLE
+  * [[VUID-vkSetDebugUtilsObjectNameEXT-device]]
+    If the object referred to by pname:pNameInfo->objectHandle was
+    created, allocated or retrieved from a slink:VkDevice,
+    pname:device must: be the device the object was created,
+    allocated or retrieved from.
+  * [[VUID-vkSetDebugUtilsObjectNameEXT-device]]
+    If pname:pNameInfo->objectType is ename:VK_OBJECT_TYPE_INSTANCE or
+    ename:VK_OBJECT_TYPE_PHYSICAL_DEVICE, pname:device must: have been
+    created from the object referred to by pname:pNameInfo->objectHandle.
+  * [[VUID-vkSetDebugUtilsObjectNameEXT-device]]
```

Comment:
All of these VUID's are the same.  I'm not sure that's allowed.  I think you need them generated so they have unique values

Reply:
It seems I misread the documentation in vuid.txt, I thought this was the required header in order to generate the VUIDs for merging. Just removing the `[[VUID-*]]` blocks (while keeping the asterisk before the statement) would work?

Reply:
That's right. If you will point to the misleading statement I'll attempt to clarify it.

Reply:
Upon rereading, everything was pretty obvious, I don't think there is a need to clarify it :)
I think I can also remove the draft status now. Thanks for helping!